cmd/mapcidr/main_test.go

```
@@ -1,12 +1,38 @@
 import (
+	"os"
 	"sync"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 )
 
+func TestOutputItemsWritesToFile(t *testing.T) {
+	f, err := os.CreateTemp(t.TempDir(), "out-*")
+	require.NoError(t, err)
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)